.github/workflows/delta-conversion-ci.yml

```diff
@@ -68,6 +68,7 @@ jobs:
   delta-conversion-scala-2-12-tests:
     runs-on: ubuntu-22.04
     strategy:
+      fail-fast: false
```

Comment:
do we need to change fail-fast strategy?

Reply:
no, that's a temporary change (separate commit)